pjmedia/build/pjmedia.vcxproj

```diff
@@ -717,6 +717,7 @@
       <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='Release-Dynamic|ARM64'">true</ExcludedFromBuild>
     </ClCompile>
     <ClCompile Include="..\src\pjmedia\transport_udp.c" />
+    <ClCompile Include="..\src\pjmedia\txt_stream.c" />
```

Comment:
Minor: please also add the `stream_imp_common.c` to the project (with exclude from build), so it can be included in searching in files.